app/views/docs/functions-deploy.phtml

```diff
@@ -0,0 +1,198 @@
+<p>
+    Give your function a name and choose your runtime. 
+    This will create a new starter function in the current directory and also add it to your <code>appwrite.json</code> file.
+</p>
+
+<p>
+    Edit the automatically generated code and add dependencies to the dependency files of your language or framework.
+    Then, deploy the function using the following command.
+</p>
+
+<div class="ide margin-bottom" data-lang="bash" data-lang-label="Bash">
+    <pre class="line-numbers"><code class="prism language-bash" data-prism>appwrite deploy function</code></pre>
+</div>
+
+<div class="notice">
+  <h2>Overwrite Warning</h2>
+    <p>
+        If you made changes in the Appwrite console that is different from your <code>appwrite.json</code>, 
+        using the CLI deploy command will overwrite your console changes, such as execution schedule or permissions.
+        Update your <code>appwrite.json</code> manually before deploying to avoid overwriting changes.
+    </p>
+</div>
+
+<h3>Manual Deployment</h3>
+<p>You can also upload your functions to be deployed using the Appwrite console. The example below shows a simple Node.JS function, but the same idea applies to any other language.</p>
+
+<div class="ide margin-bottom">
+    <pre class="line-numbers"><code class="prism" data-prism>.
+├── package.json
+└── index.js
+</code></pre>
+</div>
+
+<p>First, navigate inside the folder that contains your dependency file. Package your code files into the <code>.tar.gz</code> format with this tar command:</p>
+
+<ul class="phases clear" data-ui-phases>
+    <li>
+        <h3>Unix</h3>
+
+        <div class="ide margin-bottom" data-lang="bash" data-lang-label="Bash">
+            <pre class="line-numbers"><code class="prism language-bash" data-prism>tar --exclude code.tar.gz -czf code.tar.gz .</code></pre>
+        </div>
+    </li>
+    <li>
+        <h3>CMD</h3>
+
+        <div class="ide margin-bottom" data-lang="bash" data-lang-label="CMD">
+            <pre class="line-numbers"><code class="prism language-bash" data-prism>tar --exclude code.tar.gz -czf code.tar.gz .</code></pre>
+        </div>
+    </li>
+    <li>
+        <h3>PowerShell</h3>
+
+        <div class="ide margin-bottom" data-lang="bash" data-lang-label="PowerShell">
+            <pre class="line-numbers"><code class="prism language-bash" data-prism>tar --exclude code.tar.gz -czf code.tar.gz .</code></pre>
+        </div>
+    </li>
+</ul>
+
+<p>Next, navigate to your Appwrite console and upload the function.</p>
+
+<ol class="margin-top margin-bottom-large text-size-normal">
+  <li class="margin-bottom-small">Navigate to the function you want to deploy.</li>
+  <li class="margin-bottom-small">Click <i class="icon-plus"></i> <b>Create deployment</b>.</li>
+  <li class="margin-bottom-small">Select the <b>Manual</b> tab.</li>
+  <li class="margin-bottom-small">Input the entry point of your function under <b>Entrypoint</b>. For the example above, it would be <code>index.js</code>.</li>
```

Comment:
Let's also mention that users can also specify build commands.

Reply:
Good point